server/controllers/AuthController.js

```diff
@@ -49,11 +49,11 @@ class AuthController {
     });
     createUser.then((result) => {
       if (result.rowCount <= 0) {
-        return res.status(503).json({ status: 503, error: 'Database connection failed'});
+        return res.status(508).json({ status: 508, error: 'Database connection failed'});
```

Comment:
What is the relationship between rowCount <= 0 and `Database connection failed`. And would you rather think of less technical terms for describing your error messages?